### PR TITLE
Update thecustomizewindows.com.xml

### PR DESCRIPTION
**Pull request by AbhishekGhosh**
Should show as compatible to merge...

### CODE REVIEW

**Comment by AbhishekGhosh** [comment]
See here - https://github.com/EFForg/https-everywhere/pull/1639

File: src/chrome/content/rules/thecustomizewindows.com.xml [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+		<target host="www.thecustomizewindows.net" />
+
+	<!-- Akamai CDN -->
+
+		<securecookie host="^a248\.e\.akamai\.net$" name=".+"/> 
+		
+	<!-- allow trusted services -->
+	
+		<securecookie host="^cdn\.goroost\.com$" name=".+"/>
+		<securecookie host="^www\.googletagservices\.com$"
```
Comment (jsha) [inline src/chrome/content/rules/thecustomizewindows.com.xml]: It looks like your editor has wrapped some lines. Could you change the securecookie and exclusion tags to be on a single line?

For the rule tags, it's fine to be on two lines, but the `from` tag should be aligned with the `to` tag.

Comment (AbhishekGhosh) [inline src/chrome/content/rules/thecustomizewindows.com.xml]: Yes, `nano` made 29 lines to 35 lines ... it is a risky editor, really ...

**Comment by reedy** [comment]
Closing in favour of #1655

**Pull request closed without merging**